server.go

```
@@ -5573,3 +5584,60 @@ func (s *server) AttemptRBFCloseUpdate(ctx context.Context,
 
 	return updates, nil
 }
+
+// getNodeAnnouncementFields populates the alias, color, and addresses. This
+// is used during startup to construct the initial node announcement.
+func getNodeAnnouncementFields(ctx context.Context, cfg *Config, s *server,
+	serializedPubKey [33]byte) (color.RGBA, string, []net.Addr, error) {
+
+	var (
+		color color.RGBA
+		alias = cfg.Alias
+		addr  []net.Addr
+		err   error
+	)
+
+	// Try to parse the color from config. We will update this later if the
+	// config color is not changed from default (#3399FF).
+	color, err = lncfg.ParseHexColor(cfg.Color)
+	if err != nil {
+		srvrLog.Errorf("unable to parse color: %v\n", err)
+
+		return color, "", nil, err
```

rather just return a decorated error - it will get logged higher up